k8s/production/custom/webhook-handler/deployments.yaml

```diff
@@ -0,0 +1,190 @@
+              memory: 1G
+            limits:
+              cpu: 1600m
+              memory: 1G
+          ports:
+            - containerPort: 8080
+          env:
+            - name: GITLAB_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-secrets
+                  key: gitlab-endpoint
+            - name: GITLAB_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-handler
+                  key: gitlab-token
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-secrets
+                  key: secret-key
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-secrets
+                  key: sentry-dsn
+            - name: DB_NAME
+              value: analytics
+            - name: DB_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-handler
+                  key: analytics-postgresql-host
+            - name: DB_USER
+              value: postgres
+            - name: DB_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-handler
+                  key: analytics-postgresql-password
+            - name: OPENSEARCH_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: opensearch-secrets
+                  key: opensearch-endpoint
+            - name: OPENSEARCH_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: opensearch-secrets
+                  key: opensearch-username
+            - name: OPENSEARCH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: opensearch-secrets
+                  key: opensearch-password
+            - name: CELERY_BROKER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: webhook-secrets
+                  key: celery-broker-url
+      nodeSelector:
+        spack.io/node-pool: base
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook-handler-worker
+  namespace: custom
+  labels:
+    app: webhook-handler-worker
+    svc: web #TODO
+spec:
+  selector:
+    matchLabels:
+      app: webhook-handler-worker
+      svc: web # TODO
+      # TODO: evaluate
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: webhook-handler-worker
+        svc: web # TODO
+    spec:
+      restartPolicy: Always
+      serviceAccountName: webhook-handler  # TODO yes?
+      containers:
+        - name: webhook-handler-worker
+          image: ghcr.io/spack/upload-build-timings:0.0.5
+          command: ["celery", "-A", "analytics.celery", "worker", "-l", "info"]
+          imagePullPolicy: Always
+          resources:
+          # TODO: evaluate
+            requests:
+              cpu: 350m
+              memory: 1G
```

Comment:
I think these make sense. With the default concurrency of 4, we can potentially have 4 job logs loaded into memory at once on a single worker, which seems like 1GB of memory should always be able to handle.